crates/storage/db/src/tables/mod.rs

```diff
@@ -148,14 +149,19 @@ table!(
 
 table!(
     /// (Canonical only) Stores the transaction body for canonical transactions.
-    ( Transactions ) TxNumber | TransactionSigned
+    (  Transactions ) TxNumber | TransactionSigned
 );
 
 table!(
     /// Stores the mapping of the transaction hash to the transaction number.
     ( TxHashNumber ) TxHash | TxNumber
 );
 
+table!(
+    /// Stored the range of transaction numbers that belong to the block.
+    ( TransactionBlock ) TxNumber | BlockNumber
```

Comment:
nit: I think `CumulativeTxCount` is a bit more clear on what is stored in here.

```suggestion
    /// Stores the cumulative number of transactions for each block.
    ///
    /// The key is the highest transaction ID in the block.
    ( TransactionBlock ) TxNumber | BlockNumber
```

Reply:
More on this here: https://github.com/paradigmxyz/reth/issues/1699
It is not `ComulativeTxCount` but tx to block range. as if block does not have tx it will not have entry.

Reply:
Ok sounds good, but I would like a bit more info in the docs if possible!